applications/IgaApplication/tests/linear_scordelis_p4_nCP10/shell_5p_project_parameters.json

```diff
@@ -0,0 +1,117 @@
+        "max_iteration": 100,
+        "solver_type": "static",
+        "linear_solver_settings": {
+            "solver_type": "amgcl",
+            "max_iteration": 500,
+            "tolerance": 1E-09,
+            "scaling": false,
+            "verbosity": 1
+        },
+        "auxiliary_variables_list": [],
+        "auxiliary_dofs_list": [],
+        "auxiliary_reaction_list": [],
+        "multi_point_constraints_used": false
+    },
+    "modelers": [{
+            "modeler_name": "CadIoModeler",
+            "Parameters": {
+                "echo_level": 0,
+                "cad_model_part_name": "IgaModelPart",
+                "geometry_file_name": "linear_scordelis_p4_nCP10/geometry.cad.json"
+            }
+        },
+        {
+            "modeler_name": "IgaModeler",
+            "Parameters": {
+                "echo_level": 0,
+                "cad_model_part_name": "IgaModelPart",
+                "analysis_model_part_name": "IgaModelPart",
+                "physics_file_name": "linear_scordelis_p4_nCP10/physics_5p.iga.json"
+            }
+        }
+    ],
+    "processes": {
+        "dirichlet_process_list": [{
+                "kratos_module": "KratosMultiphysics",
+                "python_module": "assign_vector_variable_process",
+                "Parameters": {
+                    "mesh_id": 0,
+                    "model_part_name": "IgaModelPart.Support_3",
+                    "variable_name": "DISPLACEMENT",
+                    "value": [0, null, 0],
+                    "interval": [0, "End"]
+                }
+            },
+            {
+                "kratos_module": "KratosMultiphysics",
+                "python_module": "assign_vector_variable_process",
+                "Parameters": {
+                    "mesh_id": 0,
+                    "model_part_name": "IgaModelPart.Support_4",
+                    "variable_name": "DISPLACEMENT",
+                    "value": [0, null, 0],
+                    "interval": [0, "End"]
+                }
+            }
+        ],
+        "neumann_process_list": [{
+            "kratos_module": "KratosMultiphysics",
+            "python_module": "assign_vector_variable_to_conditions_process",
+            "Parameters": {
+                "mesh_id": 0,
+                "model_part_name": "IgaModelPart.Load_5",
+                "variable_name": "SURFACE_LOAD",
+                "value": [0, 0, -90],
+                "interval": [0, "End"]
+            }
+        }],
+        "json_check_process": [
+            {
+                "python_module": "from_json_check_result_process",
+                "kratos_module": "KratosMultiphysics",
+                "Parameters": {
+                    "check_variables": [ "DISPLACEMENT_X", "DISPLACEMENT_Z", "ROTATION" ],
+                    "input_file_name": "linear_scordelis_p4_nCP10/linear_scordelis_p4_nCP10_test_results.json",
+                    "model_part_name": "IgaModelPart",

```

Comment:
@tteschemacher can you please use a submodelpart here to reduce the file size of the result files?

Usually it is not necessary to check every single node, if sth goes wrong you will also know if you check a subset :)

Reply:
the file size is not even that heavy. However, it seems that the whole simulation is already too big for the CI.

Do all application have the same run time of tests? On my machine it takes only very short in comparison to the structural mechanics application.